Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -81,6 +89,165 @@ function Add-DatabaseSecurity($userName, $role, $connString)
 	Write-Success "Database security applied successfully"
 }
 
+function Invoke-Get($url, $accessToken)
+{
+    $headers = Get-Headers -accessToken $accessToken
+        
+    $getResponse = Invoke-RestMethod -Method Get -Uri $url -Headers $headers
+    return $getResponse
+}
+
+function Invoke-Post($url, $body, $accessToken)
+{
+    $headers = Get-Headers -accessToken $accessToken
+
+    if(!($body -is [String])){
+        $body = (ConvertTo-Json $body)
+    }
+    
+    $postResponse = Invoke-RestMethod -Method Post -Uri $url -Body $body -ContentType "application/json" -Headers $headers
+    Write-Success "    Success."
+    Write-Host ""
+    return $postResponse
+}
+
+function Get-Role($name, $grain, $securableItem, $authorizationServiceUrl, $accessToken){
+    $url = "$authorizationServiceUrl/roles/$grain/$securableItem/$name"
+    $role = Invoke-Get -url $url -accessToken $accessToken
+    return $role
+}
+
+function Add-Group($authUrl, $name, $source, $accessToken){
+    $url = "$authUrl/groups"
```

Comment:
Are JSON KV pairs not comma delimited?

Reply:
We are creating a hashtable here in PowerShell, then in the Invoke-Post method, we convert it to Json.